tools/gather-sysinfo/gather-sysinfo.go

```diff
@@ -149,5 +171,6 @@ func kniExpectedCloneContent() []string {
 		"/sys/class/dmi/id/product_name",
 		"/sys/class/dmi/id/product_sku",
 		"/sys/class/dmi/id/product_version",
+		"/var/log/machineinfo.json",
```

Comment:
let's remove this path from there

and let's append the container-local paths in line 102, *after* the processing done by `chrootFileSpecs`

something like

```
fileSpecs = dedupExpectedContent(fileSpecs, kniContainerLocalCloneContent())
```

this will break a little the (already rarely used) `if opts.dumpList` flow, but it's ok for now.